packages/openapi/openapi.1.0.0/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Openapi documentation infrastructure"
+description:
+  "Core types and functions for creating openapi compliant documentation using JSON schemas"
+maintainer: ["J. Aaron Pendergrass <james.pendergrass@jhuapl.edu>"]
+authors: ["J. Aaron Pendergrass <james.pendergrass@jhuapl.edu>"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/jhuapl-saralab/openapi-ocaml"
+bug-reports: "https://github.com/jhuapl-saralab/openapi-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "yojson" {>= "2.1.2"}
+  "core" {>= "v0.16.0"}
+  "ppx_yojson_conv" {>= "v0.16.0"}
+  "ppx_yojson_conv_lib" {>= "v0.16.0"}
+  "ppx_deriving" {>= "5.2.1"}
+  "odoc" {with-doc}
```

Comment:
```suggestion
  "ppx_deriving_json_schema" {with-test & post & =version }
  "odoc" {with-doc}
```
There is a circular dependency, I'll check if the above works by merging this suggestion, but it may well be that you need to move the tests using ppx_deriving_json_schema in the library itself
```
#=== ERROR while compiling openapi.1.0.0 ======================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.4.14.2 | pinned(https://github.com/jhuapl-saralab/openapi-ocaml/archive/refs/tags/v1.0.0.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/openapi.1.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p openapi -j 255 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/openapi-7-bb9a1a.env
# output-file          ~/.opam/log/openapi-7-bb9a1a.out
### output ###
# File "test/dune", line 55, characters 18-42:
# 55 |  (preprocess (pps ppx_deriving_json_schema ppx_yojson_conv)))
#                        ^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Library "ppx_deriving_json_schema" not found.
# -> required by _build/default/test/test.exe
# -> required by _build/default/test/test-schema.json
# -> required by alias test/runtest in test/dune:57
```

Reply:
Concur. Thank you. Will look into moving the tests.

Reply:
This is my first attempt at a multi-package project. I'm looking for a mechanism in dune to enable tests only for the `ppx_deriving_json_schema` package but coming up empty. Are you aware of a straightforward solution?

Reply:
You can add a `(package ppx_deriving_json_schema)` to the tests to specify for which package to run